.concourse/README.md

```diff
@@ -48,8 +48,14 @@ point to your new EKCP API endpoint.
 ## Deploy on Concourse
 
 If you wish to deploy the pipeline on Concourse, run the following
-command:
+command and use the `fly` script in this directory:
 
 ```
-fly -t target set-pipeline -c pipeline.yaml -p kubecf
+./fly -t target set-pipeline -c pipeline.yaml -p kubecf
```

Comment:
Can't we have `fly` binary in `$PATH` to maintain status quo.?

Reply:
This `fly` here is the script we include in this PR. It's wrapping fly to render the template. See the fly script below.